recipes/automake/all/test_package/conanfile.py

```diff
@@ -40,6 +40,7 @@ def _settings_build(self):
 
     def build_requirements(self):
         self.build_requires(self.tested_reference_str)
+        self.tool_requires("autoconf/2.71") # Needed for autoreconf
```

Comment:
In my testing, this was not necessary since `automake` lists `autoconf` as a requirement (https://github.com/conan-io/conan-center-index/blob/7eba669c781b0d0e445c35abc025e0a7791248a5/recipes/automake/all/conanfile.py#L37-L38) 

is that not the case?
Happy to test this - it may be a case of a missing package_type somewhere, but the intention is that the downstream consumers don't need to specify this (it also doesnt hurt).

Reply:
Hi @jcar87,
Indeed in my latest testing it does not seem to be needed so I will remove it. Thanks

P.S. I believe I am testing with autoconf configured as an "application" per #14925